lib/index.js

```diff
@@ -327,31 +349,31 @@ King = new Class({
         url = this.url( 'apis', id, 'plugins');
         debug('request url %s', url, data );
 
-        plugins_list = yield request({
+        plugins_list = yield this.request({
             method:'GET'
           , uri:url
           ,json:true
-        });
+        }, [200]);
 
-        plugins = plugins_list.body.data;
+        plugins = plugins_list.body.data || [];
```

Comment:
Hmm I'm just not able to reproduce this. Example? Better yet, can you put together a failing test that illustrates?

Also which version of kong ?